inspirehep/modules/migrator/cli.py

```diff
@@ -54,6 +57,8 @@
 def migrator():
     """Command related to migrating INSPIRE data."""
     logging.basicConfig()
+    # Disable auto-indexing receiver in migration tasks
+    models_committed.disconnect(receive_after_model_commit)
```

Comment:
Now that I think about it, shall I connect it again? Or the migrator is a one-of CLI command and future requests will have the signal connected?